src/scripts/pbs_submit.sh

```diff
@@ -223,6 +225,7 @@ jobID=`echo $jobID | awk 'match($0,/[0-9]+/){print substr($0, RSTART, RLENGTH)}'
 if [ "X$jobID" == "X" ]; then
 	rm -f $bls_tmp_file
 	echo "Error: job id missing" >&2
+  echo "Error from qsub: $jobID" >&2
```

Comment:
I think in this case you can remove the previous line. Can you also fix the indentation for both changes?